subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformWithDependenciesIntegrationTest.groovy

```diff
@@ -66,11 +65,24 @@ project(':app') {
     }
 
     dependencies {
+        // Single step transform
         registerTransform {
             from.attribute(artifactType, 'jar')
             to.attribute(artifactType, 'size')
             artifactTransform(FileSizer)
         }
+
+        // Multi step transform
+        registerTransform {
+            from.attribute(artifactType, 'jar')
+            to.attribute(artifactType, 'inter')
+            artifactTransform(FileSizer)
+        }
+        registerTransform {
+            from.attribute(artifactType, 'inter')
+            to.attribute(artifactType, 'final')
+            artifactTransform(FileSizer)
+        }
```

Comment:
I think it would be good to use something that makes sense for these transforms, something other than `FileSizer` that captures what transformation is actually happening. We should probably have two different types for the two transforms.